lib/html_proofer/cache.rb

```diff
@@ -194,7 +194,7 @@ def size(type)
                    elsif cache_version != CACHE_VERSION
                    # if cache version is newer...do something
                    else
-                     log[:internal] = log[:internal].transform_keys { |key| JSON.parse(key.to_s, symbolize_names: true) }
```

Comment:
@gjtorikian, We should still need to transform the JSON-string key back to a hash as in `internal_urls`. The initial `.to_s` however would not be needed.